microsoft/iot-cardboard-js
# Fix for recently added twin aliases not showing in twin properties list or intellisense without saving behavior

### Summary of changes 🔍 
- Fixed the need of saving behavior for aliased twin properties show up in twin properties dropdown in Status tab and intellisense used in Alerts and Widgets tabs
- Changed some adapter methods and hooks to be able to get the twin (linked or aliased) properties from the **current** elements from state of the behavior (it used to read the elements from config file to retrieve twins and their properties even if the behavior is in edit mode where you can update the twin aliases of "selected elements" or even select/unselect the elements).
- Added small checks and messages but I am going to send a separate PR for validation and error checks for TwinsTab and TwinAliasForm.

<img width="313" alt="image" src="https://user-images.githubusercontent.com/45217314/163071397-d760963d-6c82-4e17-88b5-2d2f991eb7ba.png">
<img width="341" alt="image" src="https://user-images.githubusercontent.com/45217314/163071456-03032de0-b116-4a04-9a6d-85aec774418b.png">
<img width="314" alt="image" src="https://user-images.githubusercontent.com/45217314/163071522-56ba02d4-e09e-4950-a6e2-e589a5c66959.png">
<img width="311" alt="image" src="https://user-images.githubusercontent.com/45217314/163071563-05ca1244-2f14-45ea-8ee1-3264f7a07536.png">


### Testing 🧪
1. After selecting a behavior in the builder, go to Twins tab and create a new alias or add from existing ones
2. Switch to Status tab to see if the newly added alias is showed up in the "Properties" dropdown
3. Switch to Alerts tab to see if you can use this new alias and properties of aliased twins in intellisense for trigger expression
4. Switch to Widgets tab and add a new Link widget and  see if you can use this new alias and properties of aliased twins in intellisense for url text

### Checklist ✔️
